src/lighteval/metrics/metrics_sample.py

```diff
@@ -580,7 +580,7 @@ def compute(self, golds: list[str], predictions: list[str]) -> dict:
             hlog_warn("The first metric computation step might be a bit longer as we need to download the model.")
             # We only initialize on first compute
             self.bert_scorer = BERTScorer(
-                model_type="microsoft/deberta-large-mnli", lang="en", rescale_with_baseline=True, num_layers=9
```

Comment:
Please keep the current default as is, so you don't change silently how the function work for other users. 
Idea is that if you want to use this class differently, you can simply create another BERTScorer and pass it at the initialisation of a new custom metric (you can take a look at the readme for custom metroc)

Reply:
Sounds good and thanks for the pointer on custom metrics!